### PR TITLE
Cockpit 148

### PR DESCRIPTION
**Pull request by martinpitt**
Release is done, so good to land once review is done.

### CODE REVIEW

File: content/cockpit-148.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+### Update Cluster/Registry design for image streams
+
+The design of the image streams on the Cluster and Registry pages has been
+updated to match the design of the current OpenShift Web Console.
+
+![Image Stream Tag Design](images/registry-imagestreams-design.png)
+
+### Delete Kubernetes session tokens when logging out of Cockpit
+
+In situations where Cockpit is being used on a public computer with multiple
```
Comment (stefwalter) [inline content/cockpit-148.md]: This is not about "Cockpit" This is about the "Registry Console"

File: content/cockpit-148.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+Internet Explorer as well.
+
+### Update Cluster/Registry design for image streams
+
+The design of the image streams on the Cluster and Registry pages has been
+updated to match the design of the current OpenShift Web Console.
+
+![Image Stream Tag Design](images/registry-imagestreams-design.png)
+
+### Delete Kubernetes session tokens when logging out of Cockpit
```
Comment (stefwalter) [inline content/cockpit-148.md]: This doesn't apply to Cockpit. It only applies to the "Registry Console" so

"Delete Kubernetes session tokens when logging out of the Registry Console or Dashboard"

Comment (martinpitt) [inline content/cockpit-148.md]: Oh, thanks. I'll update that on the GitHub release page too.

File: content/cockpit-148.md (hunk 1 above)
Comment (martinpitt) [inline content/cockpit-148.md]: Fix pushed.

**Pull request merged**